pallets/liquid-staking/src/lib.rs

```diff
@@ -470,12 +472,6 @@ pub mod pallet {
             }
 
             T::Assets::burn_from(Self::liquid_currency()?, &who, liquid_amount)?;
-            StakingPool::<T>::try_mutate(|b| -> DispatchResult {
-                *b = b
-                    .checked_sub(asset_amount)
-                    .ok_or(ArithmeticError::Underflow)?;
-                Ok(())
-            })?;
```

Comment:
2/3
now when user unstake, we only decrease `total_issuance of liquid_currency`, do not decrease `StakingPool` anymore.